frontend/src/views/Listen.vue

```diff
@@ -4,13 +4,17 @@
     <div class="pb-3">
       <h3>{{ $title }}</h3>
     </div>
-    <v-card color="#d9edf7">
+    <v-card color="#d9edf7" :max-width="isInternetExplorer ? '90vw' : '100%'">
       <v-card-title primary-title>
         <div>
           <h3 class="headline mb-0">Audio Player</h3>
         </div>
       </v-card-title>
-      <v-card-title primary-title>
+      <v-alert v-if="isInternetExplorer" :value="true">
+        FLAC files are not supported in Internet Explorer. Please download the
+        file.
+      </v-alert>
```

Comment:
Maybe we can use https://www.npmjs.com/package/caniuse-lite for this, not hard-coding IE here?

Reply:
I think this needs to be paired with the browser and version of the browser to give accurate results. Maybe we can use https://github.com/lancedikson/bowser to get the browser?

I think we might be able to use https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canPlayType even though some browsers that support it return an empty string the browsers that we are aiming to support all return the expected result from `canPlayType` https://caniuse.com/#feat=flac

Reply:
Let's use `canPlayType`, it's more suitable for us than pulling another dependency.